docs/migration-v1-v2.md

```diff
@@ -0,0 +1,14 @@
+## Migration Notes for Dynamic Subnets v2.0
+
+### &nbsp; Breaking change: S3 cache bucket
+
+The S3 bucket resource was replaced with an *(S3 module)[https://github.com/cloudposse/terraform-aws-s3-bucket]*.
+* Do not upgrade if using Terraform older than 1.0.
+* Do not upgrade if you are using AWS provider older than 4.22.
+
+There are no changes in input and output variables.
+
+### &nbsp; Upgrade instructions
+The upgrade is two faceted. 
+1. Update terraform to a version equal to or newer than version 1.0. You can follow the **[Upgrading to Terraform v1.0](https://www.terraform.io/language/upgrade-guides/1-0)** guide.
+2. Update the AWS provider to the latest version. You can follow the S3 bucket modules **[Upgrading to v2.0](https://github.com/cloudposse/terraform-aws-s3-bucket/wiki/Upgrading-to-v2.0)** guide.
```

Comment:
```suggestion

If you are using Terraform v1.0 or later, upgrading to v2 of this module without taking any other steps will delete your existing S3 bucket and create a new one. Since the S3 bucket is only used for caching data, this may be acceptable, and is certainly the easiest path forward.

If you want to preserve the existing S3 bucket and its data, you will need to use `terraform state mv` to move the existing resources to their new "addresses" in the Terraform state. You can find the details of how to do this in the S3 bucket module's **[Upgrading to v2.0](https://github.com/cloudposse/terraform-aws-s3-bucket/wiki/Upgrading-to-v2.0)** guide.
```

Reply:
@Nuru , can you take a look?

Reply:
@denissimonovski I think it is necessary, because the bucket module and the resource this PR replaces it with had `force_destroy` set to true.